src/sentry/static/sentry/app/components/menuItem.tsx

```diff
@@ -3,7 +3,22 @@ import React from 'react';
 import {Link} from 'react-router';
 import classNames from 'classnames';
 
-class MenuItem extends React.Component {
+type Props = {
+  header?: boolean;
+  divider?: boolean;
+  title?: string;
+  onSelect?: (eventKey: any) => void;
```

Comment:
@MeredithAnya can we do better than `any` for the type of the `eventKey`?

Reply:
from the usage it _seemed_ like we don't really care what it is. Also we (@lynnagara and I) actually debated taking this prop (`eventKey`) out all together since it doesn't seem like we need to use it in the way we do now but since we use this component in a lot of places felt safer to first just convert to ts